app/helpers/errorHelpers.js

```diff
@@ -69,6 +69,11 @@ const validateAnswerField = (value, validationType, details, payload) => {
       const { max } = details
       return ([value].flat().length <= max)
     }
+
+    case 'CONFIRM_EMAIL': {
```

Comment:
I suggest reusing the CONFIRMATION_ANSWER case from this[ similar PR](https://github.com/DEFRA/ffc-grants-cattle-housing-web/pull/6) as it covers any field that needs to match with another.